android/app/src/main/java/org/uwaterloo/subletr/pages/home/list/components/ListingPost.kt

```diff
@@ -0,0 +1,225 @@
+package org.uwaterloo.subletr.pages.home.list.components
```

Comment:
Small code refactoring. Copy paste from the HomeListChildView.

Reply:
Nice, looks good